kubernetes-elasticsearch-cluster/deploy.sh

```diff
@@ -3,11 +3,45 @@
 set -o errexit
 set -o nounset
 
-kubectl create -f es-discovery-svc.yaml
-kubectl create -f es-svc.yaml
-kubectl create -f es-master.yaml
-kubectl rollout status -f es-master.yaml
-kubectl create -f es-client.yaml
+if (( $# != 1 ))
+then
+  echo "Usage: kubernetes-elasticsearch-cluster/deploy.sh <dataset>"
+  echo "  where <dataset> is the name of a directory in dataset_config/"
+  echo "Run this script from project root"
+  exit 1
+fi
+
+dataset=$1
+java_opts=$(jq --raw-output '.es_java_opts' dataset_config/${dataset}/deploy.json)
+
+cd kubernetes-elasticsearch-cluster
+cp es-data-stateful.yaml es-data-stateful-deploy.yaml
+# If a custom es_java_opts is set, replace it in the deploy yaml file.
```

Comment:
Please sanity check the following:
- No es_java_opts line in deploy.json
- es_java_opts = ""
- es_java_opts = "-Xms10g -Xmx10g"  (You can confirm heap size with http://localhost:9200/_cluster/stats?human&pretty)

Reply:
Done (updated to support the "" case).